src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py

```diff
@@ -207,7 +207,7 @@ def call_aladdin_service(command, parameters, version):
 
     if subscription_id and correlation_id:
         context = {
-            "sessionId": correlation_id,
+            "correlationId": correlation_id,
```

Comment:
How do you deal with the requests sent by older extensions that still use `sessionId`?

Reply:
The service that was just deployed today supports both the `sessionId` parameter and the `correlationId` parameter.